docker/apply/views/help/privacy.html

```diff
@@ -13,34 +8,49 @@ <h1 class="govuk-heading-xl">Privacy Notice</h1>
         <li>
           <h2 class="govuk-heading-l">Introduction</h2>
           <p class="govuk-body">
-            This privacy notice applies to anyone who registers to be included as a provider in the
-            ‘Find a professional service abroad’ lists. It explains how we use your data in this service.
+            This privacy notice applies to anyone who registers to be included
+            as a provider in the ‘Find a professional service abroad’ lists. It
+            explains how we use your data in this service.
           </p>
           <p class="govuk-body govuk-!-margin-bottom-0">
             Please also see the central
-            <a href="https://www.gov.uk/government/publications/fcdo-privacy-notice-consular-services-in-the-uk-and-at-british-embassies-high-commissions-and-consulates-overseas"
-               target="_blank">
-              consular services privacy notice (opens in new tab)</a>
+            <a
+              href="https://www.gov.uk/government/publications/fcdo-privacy-notice-consular-services-in-the-uk-and-at-british-embassies-high-commissions-and-consulates-overseas"
+              target="_blank"
+            >
+              consular services privacy notice (opens in new tab)</a
```

Comment:
Formatting here is a bit funny. Not crucial but we can create a ticket to look into this.

Reply:
looks like a victim of the prettier git hook formatting.  Only minor and not a real issue as such so we can fix this with your subsequent PR to set the line length to 120 chars.